cloudpickle_generators/__init__.py

```diff
@@ -60,6 +66,30 @@ def _fill_generator(gen, lasti, f_locals, frame_data):
     return gen
 
 
+def _fill_coroutine(coro, lasti, f_locals, frame_data):
```

Comment:
could we make a helper that just takes the frame instead of the coro/asyncgen/gen, and then these wrappers can just be:
```python
def _fill_coroutine(coro, *args):
    _fill_helper(coro.cr_frame, *args)
    return coro
```
or something like that, maybe with `args` actually expanded

Reply:
Yeah, makes sense.